shorebird_code_push/README.md

```diff
@@ -34,35 +34,36 @@ this:
 // Import the library
 import 'package:shorebird_code_push/shorebird_code_push.dart';
 
-// Create an instance of the ShorebirdCodePush class
-final shorebirdCodePush = ShorebirdCodePush();
+// Create an instance of the updater class
+final updater = ShorebirdUpdater();
 
 // Launch your app
-void main() {
-  runApp(const MyApp());
-}
+void main() => runApp(const MyApp());
 
 // [Other code here]
 
 class _MyHomePageState extends State<MyHomePage> {
   @override
-  void initState() {
+  Future<void> initState() async {
```

Comment:
I'm not sure if async initState is kosher?

Reply:
yeah good call changed back to using .then